eagle-stores/public/index.html

```diff
@@ -14,6 +14,14 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <link rel="stylesheet" href="https://eagleui.netlify.app/css/component.css">
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css"
+    integrity="sha512-PgQMlq+nqFLV4ylk1gwUOgm6CtIIXkKwaIHp/PAIWHzig/lKZSEGKEysh0TCVbHJXCLN7WetD8TFecIky75ZfQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
you can also add comments like which link is for component library and which is the link for google fonts.

Reply:
Done.